src/main/java/com/elixrlabs/doctorpatientmanagementsystem/rest/controller/patient/PatientDeletionController.java

```diff
@@ -26,7 +23,7 @@ public PatientDeletionController(PatientDeletionService patientDeletionResourceS
     }
 
     @DeleteMapping(ApiConstants.PATIENT_ID)
-    public ResponseEntity<BaseResponse> ResponseDto(@PathVariable String patientId) throws Exception {
+    public ResponseEntity<BaseResponse> deletePatientController(@PathVariable String patientId) throws Exception {
```

Comment:
```suggestion
    public ResponseEntity<BaseResponse> deletePatientById(@PathVariable String patientId) throws Exception {
```
This is a method name not a class name.

Reply:
changed method name from deletePatientController to deletePatientId